File: Corpo.Web/ClientApp/src/app/domain/wod-member.ts
import { WodGroupMember } from "./wod-group-member";
import { IntensityType } from "./wod/periodization";

export class WodMember {
  id?: number;
  wodGroupsMember: WodGroupMember[]=[];
  memberId: number;
  detail: string;
  goal?: string;
  periodizationId: number;
  weekNumber: number;
  wodNumber: number;
  attended: string;
  rate: number;
  intensityType: IntensityType;
  intensity: number
}

File: Corpo.Domain/Models/WodMember.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Corpo.Domain.Models
{
    public class WodMember
    {
        public int Id { get; set; }
        public string Goal { get; set; }
        public virtual List<WodGroupMember> WodGroupsMember { get; set; }
        public int MemberId { get; set; }
        public int WodNumber { get; set; }
        public int PeriodizationId { get; set; }
        public int WeekNumber { get; set; }
        public string Detail { get; set; }
        public int Rate { get; set; }
        public string Attended { get; set; }
        public int Rest { get; set; }
        public IntensityType IntensityType { get; set; }
        public int Intensity { get; set; }
        public DateTime? ShiftDate { get; set; }
    }
}
